src/content/structured/accessibility/needs/invisible.mdx

```diff
@@ -86,17 +91,52 @@ As well as widespread pain, people with FMS can also experience increased sensit
   .
 </p>
 
-## Making things accessible
+## How to design for invisible illnesses
 
-In addressing some of the design issues below, we make it easier for everyone to use our apps. Thinking about the [design considerations for neurodiversity](/accessibility/needs/neurodiversity) can help people who have invisible illnesses.
+In addressing some of the design tips below, we make it easier for everyone to use our apps.
 
-### Anxiety and stress-free design
+Thinking about the [design considerations for neurodiversity](/accessibility/needs/neurodiversity) can help people who have invisible illnesses.
 
 Some things we include in apps increase anxiety and, for some people, the impact can be significant.
 
-Consider:
-
-- Error messages should explain how to fix the problem and not blame the user.
-- [Motion/animation](/styles/motion) or information should only appear when the user expects it.
-- Popups that disappear too quickly before they can be read, like [toasts](/components/toast).
-- Use simple [layouts](/styles/layout-spacing) as complicated, dense layouts can overload users with information.
+### Error messages
+
+Any error messages on a page should explain how to fix the problem without blaming the user. Being blamed can increase anxiety and stress which may affect other invisible needs.
+Ensure any error messages explain what needs to be done to fix the error.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={invisibleAccessibilityFig4}
+    imageAlt="A graphic of an error message and a submit button where the error asks the user to check data. The error states that the error is with the form, not with the user."
+    state="good"
+    caption="Error messages should advise people on how to fix the issue rather than blaming them."
+  />
+  <DoDontCaution
+    imageSrc={invisibleAccessibilityFig5}
+    imageAlt="A graphic of an error message and a submit button where the error blames the user for the submission failing."
+    state="bad"
+    caption="An error message that blames the person rather than telling them how to fix it."
```

Comment:
Suggest the caption should be changed to something like "Error messages should not blame the person or fail to tell them how to fix the issue."